app/src/pages/project/ProjectPage.tsx

```diff
@@ -109,18 +119,35 @@ export function ProjectPageContent({
   );
   const [spansQueryReference, loadSpansQuery, disposeSpansQuery] =
     useQueryLoader<ProjectPageSpansQueryType>(ProjectPageSpansQuery);
+  const [sessionsQueryReference, loadSessionsQuery, disposeSessionsQuery] =
+    useQueryLoader<ProjectPageSessionsQueryType>(ProjectPageSessionsQuery);
   const onTabChange = useCallback(
     (index: number) => {
       if (index === 1) {
+        disposeSessionsQuery();
         loadSpansQuery({
           id: projectId as string,
           timeRange: timeRangeVariable,
         });
+      } else if (index === 2) {
+        disposeSpansQuery();
+        loadSessionsQuery({
+          id: projectId as string,
+          timeRange: timeRangeVariable,
+        });
```

Comment:
interesting, why do we not need a loadTracesQuery in here is it cause traces are loaded in the project page query up above? And you can't access this page on the spans or sessions tab on first mount (it always starts on traces tab?)